File: Sche_Treatment_Project/src/Models/Calendars/CalendarResponse.ts
import { DoctorResponse } from "../Doctors/DoctorResponse";
import { GroupTime } from "../Model";

export class CalendarResponse {
  id: string;
  calendarDate: string;
  doctor: DoctorResponse;
  groupTime: GroupTime;
  constructor(
    id: string,
    calendarDate: string,
    doctor: DoctorResponse,
    groupTime: GroupTime
  ) {
    this.id = id;
    this.calendarDate = calendarDate;
    this.doctor = doctor;
    this.groupTime = groupTime;
  }
}

File: Sche_Treatment_Project_API/src/main/java/st/hcmuaf/edu/vn/sche_treatment_project_api/response/calendar/CalendarResponse.java
package st.hcmuaf.edu.vn.sche_treatment_project_api.response.calendar;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import st.hcmuaf.edu.vn.sche_treatment_project_api.model.DTO.GroupTimeDTO;
import st.hcmuaf.edu.vn.sche_treatment_project_api.response.doctor.DoctorResponse;

import java.time.LocalDate;

@NoArgsConstructor
@AllArgsConstructor
@Setter
@Getter
public class CalendarResponse {
    private String id;
    private LocalDate calendarDate;
    private DoctorResponse doctor;
    private GroupTimeDTO groupTime;
}
